production/helm/loki/templates/read/deployment-read.yaml

```diff
@@ -95,8 +95,18 @@ spec:
           {{- end }}
           securityContext:
             {{- toYaml .Values.loki.containerSecurityContext | nindent 12 }}
+          {{- with .Values.loki.livenessProbe }}
```

Comment:
For the read component, looking at #14384, we need a different endpoint for the probe at `/loki/api/v1/labels`. This endpoint is not available for write and backend.

Could we have a `.Values.read.livenessProbe` which falls back to `.Values.loki.livenessProbe`?